STIJ-306: Added w10-o1 helper class to detail layout.

delrueba opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the styleguide CHANGELOG accordingly.

delrueba committed:
STIJ-306: Added w10-o1 helper class to detail layout.

mattiasberlamont committed:
Updated Detail Layout documentation.

mattiasberlamont commented:
@delrueba I updated the Detail Layout documentation mentioning all the components that can break out of the default 7 column width with some more information, also mentioning the helper class.

delrueba committed:
Merge branch '4.x-dev' into feature/STIJ-306

delrueba committed:
STIJ-360: Added .w7-o2, .w9-o1, .w10-o1 and .w12.

delrueba commented:
@mattiasberlamont I updated the documentation with programme, image gallery (multiple) and helper classes.
Removed .search-block and .contact-box because these don't exist within this library.
